src/test/java/org/opentripplanner/model/plan/legreference/LegReferenceSerializerTest.java

```diff
@@ -3,36 +3,38 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import org.junit.jupiter.api.Test;
-import org.opentripplanner.model.FeedScopedId;
 import org.opentripplanner.model.calendar.ServiceDate;
+import org.opentripplanner.transit.model.basic.FeedScopedId;
 
 class LegReferenceSerializerTest {
 
+  private static final FeedScopedId TRIP_ID = new FeedScopedId("F", "Trip");
+  private static final ServiceDate SERVICE_DATE = new ServiceDate(2022, 1, 31);
+  private static final int FROM_STOP_POS = 1;
+  private static final int TO_STOP_POS = 3;
+  private static final String ENCODED_TOKEN =
+    "rO0ABXc0ABhTQ0hFRFVMRURfVFJBTlNJVF9MRUdfVjEABkY6VHJpcAAIMjAyMjAxMzEAAAABAAAAAw==";
+
   @Test
   void testScheduledTransitLegReferenceRoundTrip() {
-    var ref = new ScheduledTransitLegReference(
-      new FeedScopedId("Feed", "Trip"),
-      new ServiceDate(2022, 1, 31),
-      1,
-      3
-    );
+    var ref = new ScheduledTransitLegReference(TRIP_ID, SERVICE_DATE, 1, 3);
 
     var out = LegReferenceSerializer.encode(ref);
 
+    assertEquals(ENCODED_TOKEN, out);
+
     var ref2 = LegReferenceSerializer.decode(out);
 
     assertEquals(ref, ref2);
   }
-
+  // purpose
```

Comment:
This should be expanded?

Reply:
This was a mistake. I wrote it in Intellj, then deleted it - but I was at the same time running the code formatter in the background - and I guess the formatter read the file before I deleted the line, and then replaced my modified file, when done.